src/Altinn.App.Api/Controllers/OrganisationSearchController.cs

```diff
@@ -0,0 +1,53 @@
+using System.Net.Mime;
+using Altinn.App.Api.Infrastructure.Filters;
+using Altinn.App.Api.Models;
+using Altinn.App.Core.Internal.Registers;
+using Altinn.Platform.Register.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.App.Api.Controllers;
+
+/// <summary>
+/// This controller class provides Enhetsregisteret (ER) organisation search functionality.
+/// </summary>
+[AutoValidateAntiforgeryTokenIfAuthCookie]
+[ApiController]
+[Produces(MediaTypeNames.Application.Json)]
+[Consumes(MediaTypeNames.Application.Json)]
+[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+[Route("{org}/{app}/api/v1/organisations")]
+public class OrganisationSearchController : ControllerBase
```

Comment:
I just renamed the controller for looking up a person to "LookupPersonController". I felt like this is a lookup of one specific item, not necessarily a search, as we only accept a complete orgNr, not a partial one. 

If you agree, maybe rename this one as well to "LookupOrganisationController". And same with response object.

Reply:
Agreed!

Reply:
Question: To adhere to grammatical correctness, should it be `LookUpOrganisation/Person` if it is the action of looking up, or should it be `Organisation/PersonLookup` if it is the noun?

Reply:
I don't have a super strong opinion on naming/urls, but we should just keep in mind that some TE are gonna want to use these two endpoints directly through the app API, so they just need be logical and easy to understand.